src/assets/layout/home.styl

```diff
@@ -0,0 +1,3 @@
+.home
+  text-align center
+  
```

Comment:
extra line at the end of the file, remember you can configure this in your code editor

Reply:
x2